metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py

```diff
@@ -1646,27 +1666,32 @@ def get_by_id(self, id_: str) -> Optional[LookerUser]:
         return looker_user
 
     def to_platform_resource(
-        self, platform_instance: Optional[str], env: str
+        self, platform_instance: Optional[str]
     ) -> Iterable[MetadataChangeProposalWrapper]:
-        resource_urn = generate_user_id_mapping_resource_urn(
-            LOOKER, platform_instance, env
-        )
+        try:
+            new_platform_resource = LookerPlatformResource(
```

Comment:
We can use `PlatformResource.create` directly and don't really need LookerPlatformResource class